src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/srm/SymbolicRegexNode.cs

```diff
@@ -236,6 +236,8 @@ public static void Serialize(SymbolicRegexNode<S> node, System.Text.StringBuilde
             }
         }
 
+        internal SymbolicRegexNode<S> ToANF() => throw new NotImplementedException();
+
```

Comment:
I thought I'd use ToAntimirovNormalForm (ToANF) but then instead added the antimirov flag during derivative creation to avoid duplicating work afterwards. So this is never going to be used, will delete this now with a commit.